src/convertToObject.js

```diff
@@ -3,16 +3,31 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+
+  const paramArray = sourceString.split(';');
+
+  for (const param of paramArray) {
+    const key = param.split(':')[0];
+    const val = param.split(':')[1];
```

Comment:
```suggestion
    const value = param.split(':')[1];
```
It is better to use full, descriptive names, especially if it's about saving only 2-3 chars